MANIFEST.in

```diff
@@ -3,5 +3,6 @@ include README.rst
 include setup.py
```

Comment:
The PR is good, but given we use setuptools-scm, shouldn't we just drop `MANIFEST.in` altogether, since setuptools-scm by default will add all git-tracked files to the package by default?

cc @RonnyPfannschmidt